File: R/tests/testthat/test_banditNarm_2par_lapse.R
context("Test banditNarm_2par_lapse")
library(hBayesDM)

test_that("Test banditNarm_2par_lapse", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm_2par_lapse(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_banditNarm_2par_lapse.py
import pytest

from hbayesdm.models import banditNarm_2par_lapse


def test_banditNarm_2par_lapse():
    _ = banditNarm_2par_lapse(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
